examples/gogo/command/src/main/java/org/osgi/enroute/examples/gogo/command/MyCommand.java

```diff
@@ -0,0 +1,19 @@
+
+package org.osgi.enroute.examples.gogo.command;
+
+import org.osgi.enroute.examples.gogo.command.upstream.annotations.RequireCommand;
+import org.osgi.enroute.examples.gogo.command.upstream.propertytypes.Command;
+import org.osgi.service.component.annotations.Component;
+
+@Component(service = { MyCommand.class } /* You need to provide a class, without it will not be picked up. */)
```

Comment:
@timothyjward The comment will be moved to a documentation site some time.

Reply:
Great, but if the current comment is going to stay for a while then could it say:

`/* You need to provide the command as a service or it it will not be picked up. */`